swagger-spec/files/detail.yaml

```diff
@@ -3,17 +3,18 @@ get:
   description: >-
     Retrieves the details of a file (or folder)
 
-    ####Returns
+    #### Returns
 
-    Returns a JSON object with a `data` key containing the representation of the requested
-    file, if the request was successful.
+    Returns a JSON object with a `data` key containing the metadata for the requested file or folder.
+    If a metadata request is issued against a folder path the call will return the return the metadata for the folder's children.
+    To retrieve metadata for an individual folder, issue a GET request against the folder with the query parameter `info`, you will return the metadata for that folder instead of the metadata for it's children.
```

Comment:
This is a weird one to phrase; offering some suggestions. Go with whatever you think sounds most clear or some combination of them all. Need to reword "you will return" and would prefer a word other than "children" so as not to confuse with our children endpoints. 

> To retrieve the metadata for a given folder, include the `?info=` query parameter in the request against the folder. The response will return the metadata for the folder itself instead of the metadata for every object in the folder.

> If the `?info=` query parameter is present for a request against a folder, the response will contain the metadata for the folder itself instead of the metadata for the objects in the folder.